docker/gcp_pudl_etl.sh

```diff
@@ -122,6 +127,8 @@ if [[ $ETL_SUCCESS == 0 ]]; then
     if [ $GITHUB_ACTION_TRIGGER = "push" ] || [ $GITHUB_REF = "dev" ]; then
         copy_outputs_to_distribution_bucket
         ETL_SUCCESS=${PIPESTATUS[0]}
+        zenodo_data_release 2>&1 | tee -a $LOGFILE
```

Comment:
Given our current logic, the logs from the release script will show up in log file that is sent to slack but they won't show up in the log file copied to the GCS bucket. I think `copy_outputs_to_gcs` should be called towards the end of the script.

Reply:
There's a problem here now that we're doing some post-processing for distribution -- removing files we don't want to distribute, gzipping the SQLite DBs.  What we copy to GCS is for forensic purposes, which is different from what we're currently shipping to AWS, Kaggle, Zenodo, etc.

Reply:
Ah that's right. I guess it's not a huge deal if we don't include the zenodo release logs in the GCS bucket. We could also change the script to dump the outputs to GCS after the ETL runs then dump the logs after most of the post-processing/distribution logic has happened.

p.s. would love to rework this system so all of the logs are collected for us.

Reply:
Do you think it'd be better to capture the additional steps in the logs, or only save the outputs we distribute to the nightly build buckets?

Reply:
I think an overhaul of the nightly build system is definitely on the docket for next year.

Reply:
I don't think I understand your question:
> Do you think it'd be better to capture the additional steps in the logs, or only save the outputs we distribute to the nightly build buckets?

Are you asking which logs should be written to the log file?

Reply:
Nevermind I was seeing it as either or based on the ordering, but re-saving the logfile fixes everything.